aodncore/pipeline/handlerbase.py

```diff
@@ -429,6 +421,19 @@ def instance_working_directory(self):
         """
         return self._instance_working_directory
 
+    @property
+    def module_versions(self):
+        """Read-only property to access module versions
+
+        :return: module version strings for aodncore, aodndata and compliance checker modules
```

Comment:
I don't see any mention of the aodndata version here. Is it possible to include it?